static/src/javascripts/projects/common/modules/identity/api.js

```diff
@@ -257,14 +257,17 @@ define([
     };
 
     Id.saveToArticles = function (data) {
-        var endpoint = '/syncedPrefs/me/savedArticles',
+        var endpoint = '/syncedPrefs/cors/me/savedArticles',
             request = ajax({
                 url: Id.idApiRoot + endpoint,
-                type: 'jsonp',
+                type: 'json',
                 crossOrigin: true,
-                data: {
-                    body: JSON.stringify(data),
-                    method: 'post'
+                method: 'POST',
```

Comment:
not that we are fully consistent, but most methods for ajax are lowercase, `post`